blog/_posts/2020-04-15-how-to-not-break-server-startup.md

```diff
@@ -0,0 +1,93 @@
+---
+slug: "how-to-not-break-server-startup"
+date: 2020-04-14
+author: nick
+layout: blog
+title: "How to Confidently Not Break Server Startup"
+subtitle: "Introducing Tilt CI"
+image: "/assets/images/how-to-not-break-server-startup/header.jpg"
+image_caption: "I took this picture of <a href='https://www.youtube.com/watch?v=_HTIEcOm3SA'>Flynn's talk at Kubecon 2019</a>, and loved this slide so much."
+tags:
+  - ci
+  - testing
+  - kind
+---
+
+Does this story sound familiar?
+
+Make a change. Passes all tests. Push to production.
+
+Explosions!
+
+How did that happen?
+
+Faked everything out. Mismatch between dev and prod env
+
+But migrating to Kubernetes means changes to your prod env. Implies changes to server startup.
```

Comment:
idk what this "but" applies to. is it: "dev/prod mismatches already hard, k8s makes it even harder"? or "you mocked everything once upon a change, and then your prod environment changed"?

Reply:
thanks, will clarify this